engine/Shopware/Models/Document/Document.php

```diff
@@ -56,6 +56,14 @@ class Document extends ModelEntity
     private $name = '';
 
     /**
+     * Contains a unique identifier for plugins
```

Comment:
Please remove the "for plugins" part, since the introduced identifier is in no means restricted to be used by plugins only.

Reply:
Refering to my comment above, I would suggest the following comment "A freely choseable internal alias, which can be used to identify a document type independently form it's id or name, because these values may vary depending of the shop context (installed plugins) and the user customization."